glambie/const/data_groups.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class GlambieDataGroup():
+    name: str
+    long_name: str  # might add some more here later
+
+
+class GlambieDataGroups():
```

Comment:
So far this reads like read-only dictionary. If it doesn't need to be read-only, then just inherit from dict. If it does you can follow one of these examples: https://stackoverflow.com/questions/19022868/how-to-make-dictionary-read-only

But you get all the item look up stuff (and set uniqueness) for free if your dictionary is:
{ "name": GlambieDataGroup}